CHANGELOG.md

```diff
@@ -21,10 +22,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Updated material dependencies to `4.0.0-canary.735147131.0`.
 - `mwc-dialog` will now search its flattened distributed nodes and their trees
   for a focusable element.
+- **BREAKING** `mwc-slider` now emits bubbling and composed `input` and `change`
+  events from `MDCSlider:input` and `MDCSlider:change`.
```

Comment:
did you mean "events instead of" instead of "events from"?

Reply:
i do now!